docs/security/Ch-SecurityIssues.rst

```diff
@@ -0,0 +1,87 @@
+###############
+Security Issues
+###############
+
+
+
+This page describes how to report EdgeX Foundry security issues and how they are handled. 
+
+======================
+Security Announcements
+======================
+Join the edgexfoundry-announce group at: https://groups.google.com/d/forum/edgexfoundry-announce) 
+for emails about security and major API announcements.
+
+=======================
+Vulnerability Reporting
+=======================
+
+The EdgeX Foundry Open Source Community is grateful for all security reports made by users and security researchers. 
+All reports are thoroughly investigated by a set of community volunteers.
+
+.. _security_issue_template: https://github.com/edgexfoundry/edgex-go/blob/master/.github/ISSUE_TEMPLATE/4-security-issue-disclosure.md.
+To make a report, please email the private list: security-issues@edgexfoundry.org, providing as much detail as possible.
+Use the security issue template:  `security_issue_template`_.
+
+At this time we do not yet offer an encrypted bug reporting option. 
+
+
+When to Report a Vulnerability?
+=====================================
+
+- You think you discovered a potential security vulnerability in EdgeX Foundry
+- You are unsure how a vulnerability affects EdgeX Foundry
+- You think you discovered a vulnerability in another project that EdgeX Foundry depends upon (e.g. docker, MongoDB, Redis,..)
+
+When NOT to Report a Vulnerability?
+=========================================
+
+- You need help tuning EdgeX Foundry components for security
+- You need help applying security related updates
+- Your issue is not security related
+
+===============================
+Security Vulnerability Response
+===============================
+
+Each report is acknowledged and analyzed by Product Security Committee members within one week. 
```

Comment:
Is the name of the committee actually `Product Security Committee`? I thought it was `Security Issue Review` team?